model/eligibility/Eligibility.php

```diff
@@ -138,15 +139,25 @@ public function getId()
     }
 
     /**
-     * @return array
+     * @return bool
      * @throws \core_kernel_persistence_Exception
      */
+    public function isProctored()
```

Comment:
I think it would be useful to add `isProctored` or `isProctorBypassed` (based on what it represents) class property with open api annotations in the same way as testTakers, testCenter, delivery. We use this class as a reference in API documentation in REST controllers.